Base/Python/slicer/__init__.py

```diff
@@ -36,11 +36,19 @@ def _createModule(name, globals, docstring):
 except ImportError:
   available_kits = []
 
+import string, os, sys
+standalone_python = "python" in string.lower(os.path.split(sys.executable)[-1])
```

Comment:
if this is not expected to be part of the public api, either start the variable with `_` or add `del standalone_python` at the end of the file

Reply:
ok, done.